lua/lspconfig/emmet_ls.lua

```diff
@@ -0,0 +1,27 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'emmet_ls'
```

Comment:
I followed the name suggested in https://github.com/aca/emmet-ls, but this kind of deviates from the convention of not having it snake-cased. Should this change to `emmetls` perhaps?